src/CppUTestExt/IEEE754ExceptionsPlugin.cpp

```diff
@@ -71,7 +71,7 @@ void IEEE754ExceptionsPlugin::enableInexact()
 
 bool IEEE754ExceptionsPlugin::checkIeee754ExeptionFlag(int flag)
 {
-    return fetestexcept(flag);
+    return !! fetestexcept(flag);
```

Comment:
It would appear that I simply forgot to make the pull request to fix #916 :-(. It was supposed to be
```c++
return (fetestexcept(flag) != 0);
```